### PR TITLE
[REF] partner credit limit: HU#1038, CA#1, 21,  TASK# 5139

### PR DESCRIPTION
**Pull request by hugho-ad**
image

image

https://github.com/Vauxoo/yoytec/pull/1038

### CODE REVIEW

**Comment by hugho-ad** [comment]
@gquilarque can you review please

**Comment by gquilarque** [comment]
+1 Functional Test

**Comment by hugho-ad** [comment]
@moylop260 can you help me to merge this please
Regards

File: partner_credit_limit/i18n/es.po [hunk 1]
```diff
@@ -40,7 +40,8 @@ msgid "Can not validate the Invoice because Partner has late payments or has exc
 "Creadit Limit : %s"
 msgstr "No se puede validar la Factura porque el cliente tiene pagos atrasados o tiene un límite de crédito excedido.\n"
 "Por favor cubrir los pagos atrasados o revisar el límite de crédito.\n"
-"Límite de Crédito : %s"
+"Límite de Crédito : %s. module: partner_credit_limi: model:ir.model,name:partner_credit_limit.model_res_compansgid \"Companiessgstr \"Compañías"
```
Comment (moylop260) [inline partner_credit_limit/i18n/es.po]: Bad po

File: partner_credit_limit/i18n/es.po [hunk 2]
```diff
@@ -148,17 +167,17 @@ msgid "Payment Terms"
...
-#. module: partner_credit_limit
 #: help:res.partner,overdue_amount:0
 #: help:sale.order,partner_overdue_amount:0
 msgid "Payment has to be made for having reached the maturity date of the obligation."
 msgstr "El pago tiene que ser hecho por haber alcanzado la fecha de vencimiento de la obligación."
 
 #. module: partner_credit_limit
+#: field:res.partner,pending_payments_ids:0
+msgid "Pending payments ids"
+msgstr "Pending payments ids"
```
Comment (moylop260) [inline partner_credit_limit/i18n/es.po]: English?

**Comment by hugho-ad** [comment]
@moylop260 thanks! comments fixeds.

Regards

**Pull request merged**